docs/src/intro-java.md

```diff
@@ -58,7 +58,7 @@ public class App {
     <dependency>
       <groupId>com.microsoft.playwright</groupId>
       <artifactId>playwright</artifactId>
-      <version>1.30.0</version>
```

Comment:
This are the docs for the next. version of playwright which is 1.30.0. Latest stable Playwright version is 1.29 which is reflected in playwright.dev. 1.29 is still not published for java but that's a different problem tracked in https://github.com/microsoft/playwright-java/issues/1159